src/CommandHandler.js

```diff
@@ -13,51 +13,38 @@ class CommandHandler {
    */
   constructor(bot) {
     /**
-     * Whether or not the commands for this handler are ready to be called.
-     * @type {boolean}
-     * @private
-     */
-    this.commandsReady = false;
-    /**
      * The bot that this CommandHandler operates for
      * @type {Genesis}
      * @private
      */
     this.bot = bot;
 
     /**
-     * The id of the owner of the bot for permissions
-     * @type {string}
-     * @private
-     */
-    this.owner = process.env.OWNER;
-
-    /**
      * Array of command objects that can be called
      * @type {Array<Command>}
      * @private
      */
-    this.commands = [];
-    this.readyCommands();
+    this.commands = this.loadCommands();
```

Comment:
Rewrite readyCommands as loadCommands so that it returns an array. The constructor won't finish until all the commands have been loaded, so `commandsIsReady` shouldn't be necessary